packages/nimble-components/src/rich-text-mention/users/tests/rich-text-mention-users.spec.ts

```diff
@@ -0,0 +1,236 @@
+import { html, repeat } from '@microsoft/fast-element';
+import { RichTextMentionUsers, richTextMentionUsersTag } from '..';
+import type { MappingUserKey } from '../../../mapping/base/types';
+import { mappingUserTag } from '../../../mapping/user';
+import { mappingTextTag } from '../../../mapping/text';
+import { Fixture, fixture } from '../../../utilities/tests/fixture';
+import { iconAtTag } from '../../../icons/at';
+import { MappingUserConfig } from '../../base/models/mapping-user-config';
+
+interface BasicUserMentionMapping {
+    key?: MappingUserKey;
+    displayName?: string;
+}
+
+describe('RichTextMentionUsers', () => {
+    let element: RichTextMentionUsers;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    async function setup(
+        mappings: BasicUserMentionMapping[],
+        pattern = ''
+    ): Promise<Fixture<RichTextMentionUsers>> {
+        return fixture<RichTextMentionUsers>(html`
```

Comment:
Do a prettier-ignore to make the template readable, this whitespace is very confusing

Reply:
Updated